### PR TITLE
Database Security: second part

### LINKED ISSUE #122: Database Security: second part
```Task``` is here: #120

### PR DESCRIPTION
**Pull request by JJerome-NM**
image

### CODE REVIEW

File: timetable-identity-access/src/main/resources/db/init.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
+create schema timetable_identity_access;
```
Comment (ArsenOOO7) [inline timetable-identity-access/src/main/resources/db/init.sql:1]: _service

Comment (ArsenOOO7) [inline timetable-identity-access/src/main/resources/db/init.sql]: _ms

Comment (ArsenOOO7) [reply timetable-identity-access/src/main/resources/db/init.sql]: or without timetable, but instead _microservice

File: timetable-identity-access/src/main/resources/db/init.sql [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+create role timetable_identity_access_role;
+
+grant connect on database pnu_system to timetable_identity_access_role;
+grant usage on schema timetable_identity_access to timetable_identity_access_role;
+grant all on schema timetable_identity_access to su_role;
+
+create user timetable_identity_access_user nosuperuser nocreatedb nocreaterole login encrypted password '12345';
```
Comment (ArsenOOO7) [inline timetable-identity-access/src/main/resources/db/init.sql]: ```identity_access_user```

File: timetable-identity-access/src/main/resources/db/init.sql [hunk 3]
```diff
@@ -0,0 +1,11 @@
+create schema timetable_identity_access;
+create role timetable_identity_access_role;
```
Comment (ArsenOOO7) [inline timetable-identity-access/src/main/resources/db/init.sql]: ```identity_access_role```

**Review by ArsenOOO7 — CHANGES_REQUESTED** [review]
Good job!

**Review by ArsenOOO7 — APPROVED** [review]

**Review by Marko-Shevchuk — APPROVED** [review]

**Pull request merged**